examples/shopping_ads/add_performance_max_retail_campaign.py

```diff
@@ -641,19 +652,49 @@ def create_and_link_image_asset(
     asset_group_asset = mutate_operation.asset_group_asset_operation.create
     asset_group_asset.field_type = field_type
     asset_group_asset.asset_group = asset_group_service.asset_group_path(
-        customer_id,
-        _ASSET_GROUP_TEMPORARY_ID,
+        customer_id, _ASSET_GROUP_TEMPORARY_ID,
     )
     asset_group_asset.asset = asset_service.asset_path(
-        customer_id, next_temp_id
+        customer_id, _NEXT_TEMP_ID
     )
     operations.append(mutate_operation)
 
-    next_temp_id -= 1
+    _NEXT_TEMP_ID -= 1
     return operations
     # [END add_performance_max_retail_campaign_8]
 
 
+# [START add_performance_max_retail_campaign_12]
+def sort_asset_and_asset_group_asset_operations(operations):
+    """Sorts a list of asset group and asset group operations.
```

Comment:
Shouldn't this say "Sorts a list of asset and asset group operations"?

Reply:
Actually it should say "Sorts a list of asset and asset group asset operations"